apps/web/src/components/SkillSummaryTable/SkillSummaryTable.tsx

```diff
@@ -0,0 +1,216 @@
+import React from "react";
+import { ColumnDef, createColumnHelper } from "@tanstack/react-table";
+import { useIntl } from "react-intl";
+import XCircleIcon from "@heroicons/react/24/solid/XCircleIcon";
+import CheckCircleIcon from "@heroicons/react/20/solid/CheckCircleIcon";
+import CheckIcon from "@heroicons/react/20/solid/CheckIcon";
+
+import {
+  commonMessages,
+  getAssessmentStepType,
+  getLocalizedName,
+  getPoolSkillType,
+  getSkillCategory,
+} from "@gc-digital-talent/i18n";
+import {
+  AssessmentStep,
+  AssessmentStepType,
+  PoolSkill,
+  PoolSkillType,
+  SkillCategory,
+} from "@gc-digital-talent/graphql";
+import { unpackMaybes } from "@gc-digital-talent/forms";
+
+import Table from "~/components/Table/ResponsiveTable/ResponsiveTable";
+import cells from "~/components/Table/cells";
+
+const columnHelper = createColumnHelper<PoolSkill>();
+
+export interface SkillSummaryTableProps {
+  poolSkills: Array<PoolSkill>;
+  assessmentSteps: Array<AssessmentStep>;
+  title: string;
+}
+
+const SkillSummaryTable = ({
+  title,
+  poolSkills,
+  assessmentSteps,
+}: SkillSummaryTableProps) => {
+  const intl = useIntl();
+
+  const CheckCircleIconElement = (
+    <CheckCircleIcon
+      data-h2-width="base(x1)"
+      data-h2-display="base(inline-block)"
+      data-h2-vertical-align="base(bottom)"
+      data-h2-margin="base(0, x.25, 0, 0) p-tablet(0, x0.5, 0, 0)"
+      data-h2-color="base(success)"
+    />
+  );
+  const XCircleIconElement = (
+    <XCircleIcon
+      data-h2-width="base(x1)"
+      data-h2-display="base(inline-block)"
+      data-h2-vertical-align="base(bottom)"
+      data-h2-margin="base(0, x.25, 0, 0) p-tablet(0, x0.5, 0, 0)"
+      data-h2-color="base(error)"
+    />
+  );
+  const CheckIconElement = (
+    <CheckIcon
+      data-h2-width="base(x1)"
+      data-h2-display="base(inline-block)"
+      data-h2-vertical-align="base(bottom)"
+      data-h2-margin="base(0, x.25, 0, 0) p-tablet(0, x0.5, 0, 0)"
+    />
+  );
+
+  const generateAssessmentStepHeader = (
+    assessmentStep: AssessmentStep,
+  ): string => {
+    if (assessmentStep.title && assessmentStep.type) {
+      return `${getLocalizedName(
+        assessmentStep.title,
+        intl,
+      )} (${intl.formatMessage(getAssessmentStepType(assessmentStep.type))})`;
+    }
+
+    if (!assessmentStep.title && assessmentStep.type) {
+      return intl.formatMessage(getAssessmentStepType(assessmentStep.type));
+    }
+
+    if (assessmentStep.title && !assessmentStep.type) {
+      return getLocalizedName(assessmentStep.title, intl);
+    }
+
+    return intl.formatMessage(commonMessages.notAvailable);
+  };
+
+  const plannedAssessmentCell = (poolSkill: PoolSkill): JSX.Element => {
+    return poolSkill.assessmentSteps && poolSkill.assessmentSteps.length > 0
+      ? CheckCircleIconElement
+      : XCircleIconElement;
+  };
```

Comment:
We will need to add some text (possibly an `aria-label`) to indicate what the icons mean to AT users.

Reply:
Like so https://github.com/GCTC-NTGC/gc-digital-talent/pull/8169/commits/5e83575c41edc596cf4e7c93020ca35b8da1ff6c ?

Reply:
That's amazing! ⭐ Thank-you 😄